code/menuui/playermenu.cpp

```diff
@@ -663,14 +663,17 @@ void player_select_button_pressed(int n)
 		break;
 
 	case SINGLE_BUTTON:
-		player_select_set_bottom_text("");
-
+		
 		Player_select_autoaccept = 0;
 		// switch to single player mode
 		if (Player_select_mode != PLAYER_SELECT_MODE_SINGLE) {
 			// play a little sound
 			gamesnd_play_iface(InterfaceSounds::USER_SELECT);
 
+			// moving this here, from outside the if, keeps bottom text from being empty by accident,
+			// when the player accidentally presses the already-lit mode
```

Comment:
A tip for writing comments, write them to be stateless to make them easier to understand without looking at the file history. I would write it something like this:
```suggestion
      // Only do this when changing modes, keeps bottom text from being empty by accident,
			// when the player accidentally presses the already-lit mode
```